static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -722,6 +732,7 @@ define([
      */
     var dfp = {
         init:           init,
+        load:           load,
```

Comment:
I might have called this `loadAds`, so it's clear what we're displaying ads that already exist, rather than kicking off our DFP dependencies.